modules/apps/export-import/export-import-api/src/main/java/com/liferay/exportimport/portlet/preferences/processor/base/BaseExportImportPortletPreferencesProcessor.java

```diff
@@ -258,6 +282,107 @@ protected void updateImportPortletPreferencesExternalReferenceCodes(
 		portletPreferences.setValues(key, newValues);
 	}
 
+	private String _getGroupExportPortletPreferencesExternalReferenceCode(
+		PortletDataContext portletDataContext, String className,
+		String externalReferenceCode) {
+
+		if (!className.equals(Group.class.getName())) {
```

Comment:
Maybe this is not needed because we are already checking it when we are calling to this method.

Reply:
I'll do the change, thanks @Mikellorza :heart_hands: